splunk_connect_for_snmp/snmp/varbinds_resolver.py

```diff
@@ -209,16 +209,25 @@ def __add__(self, other):
         new_instance = Profile(f"{self.name}:{other.name}", {})
         new_instance.varbinds_bulk = self.varbinds_bulk + other.varbinds_bulk
         new_instance.varbinds_get = self.varbinds_get + other.varbinds_get
-        new_instance.varbinds_bulk_mapping = dict(
-            self.varbinds_bulk_mapping, **other.varbinds_bulk_mapping
+        new_instance.varbinds_bulk_mapping = self.add_mappings(
+            self.varbinds_bulk_mapping, other.varbinds_bulk_mapping
         )
-        new_instance.varbinds_get_mapping = dict(
-            self.varbinds_get_mapping, **other.varbinds_get_mapping
+        new_instance.varbinds_get_mapping = self.add_mappings(
+            self.varbinds_get_mapping, other.varbinds_get_mapping
         )
         return new_instance
 
     def __repr__(self):
-        return f"Profile: {self.name}, varbinds_get: {self.varbinds_get}, varbinds_bulk: {self.varbinds_bulk}"
+        return f"Profile: {self.name}, varbinds_get: {self.varbinds_get}, varbinds_bulk: {self.varbinds_bulk}, varbinds_get_mapping: {self.varbinds_get_mapping}, varbinds_bulk_mapping: {self.varbinds_bulk_mapping}"
+
+    def add_mappings(self, dict1, dict2):
```

Comment:
can you add a comment of what does this function do? or rename variables to be more specific?

Reply:
i've added description to method